### DISCUSSION TITLE
Microsoft's lage - differentiators?

### DISCUSSION THREAD

**Discussion by Izhaki** [vercel/turborepo#468] [Help] [answered]
Just saw this for the first time:

https://github.com/microsoft/lage

Looks very much like Turborepo.

Any notable differences between the two?

(Turborepo - Go. Lage - JS.)

**Comment by seanblonien** [comment]
"Looks very much like Turbo repo" almost is an understatement, as their docs are just so nearly identical it's just so clearly copy/pasted

[Lage Pipelines](https://microsoft.github.io/lage/guide/pipeline.html) vs [Turborepo Pipelines](https://turborepo.org/docs/features/pipelines)

This [reddit post](https://www.reddit.com/r/webdev/comments/rde792/it_really_seems_like_turborepo_copied_microsoft/) is what pointed this out to me (@andrewbrey I believe?)

**Reply by andrewbrey** [reply]
Yes, that was my Reddit post, and I gave credit in there to the Reddit user who made _me_ aware of this - fortunately, one of the developers working at Nx (Jeff Cross) did the leg work of adding more attribution to the docs (and also captured even more information showing these pages and the API are copied). See https://github.com/vercel/turborepo/pull/370 which was merged a couple weeks ago now :)

**Comment by anthonyshew** [comment, marked as answer]
Looks like there was prior art at the time! Can now be found on the acknowledgments page: https://turbo.build/repo/docs/acknowledgements
